.gitpod.yml

```diff
@@ -7,9 +7,11 @@ tasks:
       export FLASK_APP=yaptide.application
       export FLASK_DEBUG=1
     init: |
-      pip install -r requirements.txt
+      export POETRY_VIRTUALENVS_CREATE=false
+      export FLASK_SQLALCHEMY_DATABASE_URI=sqlite://
```

Comment:
this seems like in-memory database, I think that file-based may be more universal for gitpod. Easier to inspect.